projects/plugins/jetpack/tests/e2e/lib/utils-helper.js

```diff
@@ -160,6 +160,31 @@ async function logAccessLog() {
 	logger.slack( { type: 'debuglog', message: apacheLog } );
 }
 
+/**
+ * Formats a given file name by replacing unaccepted characters (e.g. space)
+ *
+ * @param {string} filePath the file path. can be absolute file path, file name only, with or without extension
+ * @param {boolean} includeTimestamp if true, the current timestamp will be added as a prefix
+ * @return {string} the formatted file path
+ */
+function fileNameFormatter( filePath, includeTimestamp = true ) {
+	const parts = path.parse( path.normalize( filePath ) );
+	let fileName = parts.name;
+	const ext = parts.ext;
+	const dirname = parts.dir;
+
+	if ( includeTimestamp ) {
+		const now = new Date();
+		const formattedDate = `${ now.getFullYear() }${ now.getMonth() }${ now.getDay() }${ now.getHours() }${ now.getMinutes() }${ now.getSeconds() }${ now.getMilliseconds() }`;
```

Comment:
I got quite a weird date after the above formatting: `20212313123534`. Apparently, `getDay ` does not pad single-digits, and `getDay` actually returns the day of the week, instead of the day of the month. 

Maybe we can stick with a raw timestamp for now? Or use something like this:

```
new Date().toISOString().split( '.' )[ 0 ].replace( /:/g, '-' );
> '2021-03-10T13-43-28'
```

Reply:
My first implementation was `toISOString()`, but I wanted a shorter name. I then went with timestamp and then decided  to have the custom format just so we can visually see when the file was recorded. I knew the day is not zero padded but  I didn't find it an too important to do something about it. 

The are 2 important things here:
* use milliseconds, because we have cases when 2 screenshots are taken one after another, so we want 2 files
* use time, so we can order the files

I'll change it to use timestamp.

Reply:
Done